### PR TITLE
overview page, filter by name, sorting, details page on click

### PR DESCRIPTION
**Pull request by MrTim**
Hey people,
Creating PR even though I need to finish the styling part (will do it during weekend). Right now it looks like this  
Screenshot from 2021-05-21 20-29-05

Any feedback is welcome as usual! :smiley:

### CODE REVIEW

**Comment by MrTim** [comment]
Strange...can't assign more then one reviewer :man_shrugging: 
@saviomuc @ajinkabeer

**Review by saviomuc — APPROVED** [review]
LGTM

I was a bit confused about the need for `localeCompare`, would probably have used sort in that case, but guess you opted for it because the API is easier to understand. 
Also cool that you are using new feats like `aspect ration` and `css grids`
Should wait until styling is finished. =)

File: src/Overview/Overview.jsx [hunk 1]
```diff
@@ -0,0 +1,103 @@
+import { useState, useMemo } from "react";
```
Comment (saviomuc) [inline src/Overview/Overview.jsx:1]: Should we always use `index.js` or `index.jsx` in folders? I personally don't like repetition of `<folderName>/<folderName.js|jsx>`

Comment (saviomuc) [reply src/Overview/Overview.jsx:1]: @correttojs @ajinkabeer @MrTim

Comment (MrTim) [reply src/Overview/Overview.jsx:1]: @saviomuc yes, we should go with index.js everywhere (I missed it in here). I have the same dislike as you in here :)

**Comment by MrTim** [comment]
changed styling, to be in line with details page in another PR
Screenshot from 2021-05-24 16-28-35

**Pull request merged**